golem_cli/src/manifest.rs

```diff
@@ -35,40 +35,48 @@ pub async fn manifest_bundle(command: ManifestBundleCommand) -> Result<i32> {
 }
 
 pub async fn add_manifest_bundle(path: String) -> Result<i32> {
-    add_certs(&path).await?;
     add_whitelisted_domains(&path).await?;
+    set_rules(&path).await?;
```

Comment:
Minor aesthetics: It is probably only a matter of preference but I'd rather get rid of the `set_rules` function and would put its content directly here - this make it 0.1 sec faster to read.  
```suggestion
    set_audited_payload_rules(&path).await?;
    set_partner_rules(&path).await?;
```